_posts/en/newsletters/2024-12-13-newsletter.md

```diff
@@ -0,0 +1,462 @@
+  Gregorio Guidi, although Pickhardt arrived at it from a different
+  approach, and also appears to be confirmed in unpublished research by
+  Anastasios Sidiropoulos.
+
+  ![Example of cycles, depletion, and residual spanning tree](/img/posts/2024-12-depletion.png)
+
+  Perhaps the most notable insight provided by this result is that
+  widespread channel depletion occurs even in a circular economy where
+  there are no source nodes (i.e., net spenders) and sink nodes (i.e.,
+  net receivers).  If LN was used for every
+  payment---customer-to-business, business-to-business, and
+  business-to-worker---it would still converge on a spanning tree.
+
+  It's not clear whether nodes would want their channels to be part of
+  the residual spanning tree or not.  One one hand, that tree represents
+  the last part of the network that would still be capable of forwarding
+  payments---it's equivalent to a hub-and-spoke topology---so it may be
+  possible to charge high forwarding fees across the residual channels.
+  On the other hand, the residual channels are what's left after all
+  other channels have already collected fees to depletion.
+
+  Although a channel with higher forwarding fees is less likely to
+  become depleted (all other things being equal), the properties of
+  other channels in the same cycles strongly influence the likelihood of
+  depletion, making it challenging for a node operator to attempt to use
+  control over their own forwarding fees alone to prevent depletion.
+
+  Higher capacity channels are also less likely to become depleted than
+  lower capacity channels.  That seems obvious, but carefully
+  considering the reason for it yields a surprising insight about k>2
+  multiparty offchain protocols.  A higher capacity channel supports a
+  greater number of wealth distributions between its participants, so
+  payments through it remain feasible when equivalent payments through
+  lower capacity channels would exhaust a party's balance.  For two
+  participants, as in current-generation LN channels, each additional
+  satoshi given to capacity will increase the range of the wealth
+  distribution by one.  However, in [channel factories][topic channel
+  factories] and other multiparty constructions that allow funds to move
+  offchain between _k_ parties, each additional sat given to capacity
+  increases the range of wealth distributions by one _for each of the
+  k parties_---greatly increasing the number of feasible payments and
+  reducing the risk of depletion.
+
+  For example, consider an example of the current LN where Alice has
+  channels with Bob and Carol, who also each have a channel with each
+  other: {AB, AC, BC}.  Each channel has a capacity of 1 BTC.  In this
+  configuration, the maximum value Alice can control (and thus send or
+  receive) is 2 BTC.  The same limit applies to Bob and Carol.  The same
+  limits would also apply if the total 3 BTC was used to recreate all
```

Comment:
We say the same limit (2 BTC wealth distribution limit, I assume) applies in the channel factory example, but go on the say that 3 BTC for each participant is possible. Perhaps Im confused here.

Reply:
Added additional description that hopefully clarifies.  Thanks for letting me know it was confusing!